From 1ed00de5a9d917363c0279e3d62d14354ab8468d Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Fri, 18 Aug 2017 16:08:47 +0800
Subject: [PATCH] ARM: dts: rockchip: add reserved-memory for trust on
 rk3229-echo board

This patch reserved 14M memory zone for OPTEE side on rk3229-echo board.

Change-Id: I46de3f7163dc14361d41476daf44957184f43e89
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-echo-v10.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/rk3229-echo-v10.dts b/arch/arm/boot/dts/rk3229-echo-v10.dts
index 7c821d1696ab..e4bfeab7e1c2 100644
--- a/arch/arm/boot/dts/rk3229-echo-v10.dts
+++ b/arch/arm/boot/dts/rk3229-echo-v10.dts
@@ -55,6 +55,17 @@
 		reg = <0x60000000 0x40000000>;
 	};
 
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		trust_reserved: trust@0x68400000 {
+			 reg = <0x68400000 0xe00000>;
+			 no-map;
+		};
+	};
+
 	sdio_pwrseq: sdio-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		clocks = <&rk805 1>;
-- 
2.35.3

